mzmine-community/src/main/java/io/github/mzmine/datamodel/features/FeatureAnnotationPriority.java

```diff
@@ -43,8 +46,8 @@
  * {@link FeatureAnnotationIterator}
  */
 public enum FeatureAnnotationPriority {
-  MANUAL(ManualAnnotationType.class), LIPID(LipidMatchListType.class), SPECTRAL_LIBRARY(
-      SpectralLibraryMatchesType.class), EXACT_COMPOUND(CompoundDatabaseMatchesType.class), FORMULA(
+  MANUAL(ManualAnnotationType.class), SPECTRAL_LIBRARY(
+      SpectralLibraryMatchesType.class), LIPID(LipidMatchListType.class), EXACT_COMPOUND(CompoundDatabaseMatchesType.class), FORMULA(
```

Comment:
Ill revert this change, lipid matches > speclib for lipids

Reply:
:D oh no.
Cannot even imagine lipids should be higher than spectral library.